views/products.hbs

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="vendors/css/normalizes.css">
+    <link rel="stylesheet" href="res/css/products.css">
+    <link rel="stylesheet" href="res/css/queries.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link href="https://fonts.googleapis.com/css?family=Dosis:400,600,700" rel="stylesheet">
+    <title>SABKA BAZAAR</title>
+</head>
+
+<body>
+    <header>
+        <div class="row">
+            <nav>
+                <figure>
+                    <img src="res/img/logo.png" alt="logo" class="logo">
+                </figure>
+                <ul class="links">
+                    <li><a href="/">Home</a></li>
+                    <li><a href="/products">Products</a></li>
+                </ul>
+                <div class="cartBox">
+                    <ul class="cartBox__links">
+                        <li><a href="/signIn">Sign In</a></li>
+                        <li><a href="/signUp">Register</a></li>
+                    </ul>
+                    <div class="cart" onclick="showModalBox()">
+                        <img src="res/img/cart.svg" alt="cart">
+                        <p class="cart__items">0 items</p>
+                    </div>
+
+                </div>
+                <!-- Simulate a smartphone / tablet -->
+
+
+                <!-- Top Navigation Menu -->
+
+                <div class="topnav">
+                    <a href="javascript:void(0);" class="icon" onclick="myFunction()">
+                        <i class="fa fa-bars"></i>
+                    </a>
+                </div>
+
+                <!-- End smartphone / tablet look -->
+
+
+            </nav>
+        </div>
+    </header>
+
+    <section class="mobile-menu">
```

Comment:
Why separate menu for Mobile? Why is this not part of header. Ideally menu in header should be styled to appear as list in mobile.